integration/nwo/deploy.go

```diff
@@ -242,7 +241,11 @@ func ApproveChaincodeForMyOrg(n *Network, channel string, orderer *Orderer, chai
 				InitRequired:        chaincode.InitRequired,
 				CollectionsConfig:   chaincode.CollectionsConfig,
 				ClientAuth:          n.ClientAuthRequired,
-			})
+			}
+			if !p.DevMode {
```

Comment:
I think we should invert this to match the typical usage (i.e. all networks outside of the devmode suite don't use devmode): always set the packageID above but set it to "" when devmode is enabled. What do you think?

Reply:
good point, done.